src/app/components/vertical-icons-component.js

```diff
@@ -69,15 +73,25 @@ class VerticalIconComponent {
   setIconStatus (name, status) {
     const el = this.icons[name]
     if (!el) return
-    let statusEl = el.querySelector('i')
+    let statusEl = el.querySelector('span')
     if (statusEl) {
       el.removeChild(statusEl)
     }
     if (status.key) {
       let key = helper.checkSpecialChars(status.key) ? '' : status.key
       let type = helper.checkSpecialChars(status.type) ? '' : status.type
       let title = helper.checkSpecialChars(status.title) ? '' : status.title
-      el.appendChild(yo`<i title="${title}" class="fa fa-${key} ${css.status} text-${type}" aria-hidden="true"></i>`)
+      el.appendChild(yo`<span title="${title}" class="fa fa-${key} ${css.status} text-${type}" aria-hidden="true"></span>`)
+
+      // el.classList = "" doesn't work on all browser use instead
+      var classList = el.classList
+      while (classList.length > 0) {
+        classList.remove(classList.item(0))
```

Comment:
i guess you should remove all the classes here right?
`border-${type}` is not removed here

Reply:
hmm ok sorry for that please scratch that comment ;)

Reply:
:)